src/App.jsx

```diff
@@ -14,57 +16,55 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setSelected] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
-      <button data-cy="ClearButton" type="button" className="delete ml-3" />
-    </h1>
+  return (
+    <main className="section container">
+      <h1 className="title is-flex is-align-items-center">
+        {selectedGood !== ''
+          ? `${selectedGood} is selected`
+          : 'No goods selected'}
+        {selectedGood && (
+          <button
+            onClick={() => setSelected('')}
```

Comment:
Consider will be better if you move this logic to the helper function and use it here
```suggestion
            onClick={reset}
```